biblemulticonverter/src/main/java/biblemulticonverter/format/paratext/AbstractParatextFormat.java

```diff
@@ -347,16 +356,37 @@ protected ParatextBook exportToParatextBook(Book bk, String bibleName) {
 	}
 
 	private String internalVerseNumberToParatextVerseNumber(String internalNumber) {
-		if (LocationParser.isValidVerseId(internalNumber)) {
+		if (LocationParser.isValidVerseId(internalNumber, true)) {
 			return internalNumber;
 		} else {
-			// Try to parse the internal verse number:
-			Matcher matcher = Pattern.compile("([1-9][0-9a-z]*)(?:[,/.-]([1-9][0-9a-z]*))?").matcher(internalNumber);
+			// Try to parse the internal verse number
+			// A lot of different formats can be expected here:
+			// 5
+			// 5-6
+			// 5G
+			// 5/7
+			// 5/7/9
+			// 5.6G
+			Matcher matcher = Pattern.compile("([1-9][0-9a-zG]*)(?:([,/.-])([1-9][0-9a-zG]*))*").matcher(internalNumber);
```

Comment:
I am not 100% sure if these capturing groups inside of noncapturing groups behave as you would expect (or where it is specified in Java). If they do, please add explicit test cases for `1-4.7` and `1.4-7` (resulting in `1-4` and `1` respectively).

One special case I forgot which comes from Accordance is 1/t, refering to the title "verse" of a Psalm (if the psalm starts with verse 1 after the title verse). But rewriting this to 1 should be fine (it would also be fine to special case it to "0" or whatever is usually used in USX for this verse number).